### DISCUSSION TITLE
Welcome to Ersilia Discussions!

### DISCUSSION THREAD

**Discussion by miquelduranfrigola** [ersilia-os/ersilia#4] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yokwejuste** [comment]
Greetings! I'm Steve Yonkeu from Outreachy initial applicant list. I'm looking forward to contribute to ersilia project.

Any issues where I can start? @miquelduranfrigola

**Reply by miquelduranfrigola** [reply]
README file related issue are the easiest to start

**Comment by 0DUNTAN** [comment]
Hi, I'm Oduntan Mubarak, An  Initial applicant on Outreachy, Glad to contribute to this Project, Taking my Task one at a time, would need guidelines on Task to start with

**Reply by miquelduranfrigola** [reply]
Perhaps start with README file related issues

**Reply by 0DUNTAN** [reply]
Okay, What am I to Update on the ReadMe File sir, I am created a Readme file on the documentation file already, This is the link to the Readme i created sir https://github.com/ersilia-os/ersilia/compare/master...0DUNTAN:patch-1

**Reply by miquelduranfrigola** [reply]
Forwarding this to @GemmaTuron she takes care of readme files

**Comment by HenokB** [comment]
Hello, I'm Henok and I'm new to Ersilia. Glad to find you out here

**Reply by miquelduranfrigola** [reply]
I @HenokB nice to meet you. How can we help you?

**Comment by sofia-b-chagas** [comment]
Hi guys!

This project became one of which I'm most interested to contribute in this year's Outreachy season and since contribution period already begun, I was wondering if there are any issues open that I could adress.
I don't know if it's still required to complete the (closed, but Outreachy related) task (#36), but I think I'll go throught that just so I can get a grasp of how the model hub works. 
Is it ok?

**Comment by zheeburg** [comment]
Greetings! I'm Adedoja Isaac from Outreachy initial applicant list. I'm looking forward to contribute to ersilia project.

**Comment by miquelduranfrigola** [comment]
@GemmaTuron we are really not using GitHub discussions. Should we close this functionality?

**Comment by GemmaTuron** [comment, marked as answer]
@miquelduranfrigola 

I want to move our work to Git Discussions, perhaps I'll start anew? Apologies for not being on top of this
